### DISCUSSION TITLE
raspberry : fail to install

### DISCUSSION THREAD

**Discussion by todely** [haugene/docker-transmission-openvpn#2042] [Q&A] [answered]
Hello,
I had an old version of this docker and does not work anymore.
I have try to install and configure the latest on my raspberry but can't get succes..

I have try to start with:
docker run --cap-add=NET_ADMIN -d -v /mydirectory/:/data -e OPENVPN_PROVIDER=NORDVPN -e OPENVPN_USERNAME=myusernordvpn -e OPENVPN_PASSWORD=mypasswordnordvpn -e LOCAL_NETWORK=192.168.0.0/24 --log-driver json-file --log-opt max-size=10m --name torrent -p 9091:9091 haugene/transmission-openvpn

Docker won't start and first error on log is initial DNS resolution fail, don't know why ? :

Starting container with revision: 6922dd6b112d63e099b98165d7cadeaf411b7800
WARNING: initial DNS resolution test failed
/etc/openvpn/start.sh: line 32:    11 Aborted                 (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1
Creating TUN device /dev/net/tun
Using OpenVPN provider: NORDVPN
Running with VPN_CONFIG_SOURCE auto
Provider NORDVPN has a bundled setup script. Defaulting to internal config
Executing setup script for NORDVPN
1970-02-22 20:08:27 Checking curl installation
1970-02-22 20:08:27 Removing existing configs
1970-02-22 20:08:27 Selecting the best server...
1970-02-22 20:08:27 Searching for technology: openvpn_udp
1970-02-22 20:08:27 Unable to find a server with the specified parameters, using any recommended server
1970-02-22 20:08:27 Best server :
1970-02-22 20:08:27 Downloading config: default.ovpn
1970-02-22 20:08:27 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/.udp.ovpn
curl: (28) Timeout was reached
No VPN configuration provided. Using default.
Modifying /etc/openvpn/nordvpn/default.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
Modification: Change ca certificate path
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
Modification: Change ping options
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
sed: can't read /etc/openvpn/nordvpn/default.ovpn: No such file or directory
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
Fatal Python error: pyinit_main: can't initialize time
Python runtime state: core initialized
PermissionError: [Errno 1] Operation not permitted

Current thread 0xb6fde460 (most recent call first):
<no Python frame>
adding route to local network 192.168.0.0/24 via 172.17.0.1 dev eth0
Thu Jan  1 00:00:00 1970 disabling NCP mode (--ncp-disable) because not in P2MP client or server mode
Options error: You must define TUN/TAP device (--dev)
Use --help for more information.

**Comment by todely** [comment, marked as answer]
DNS solved with this:
https://docs.linuxserver.io/faq

Edit: rasp distri upgrade not solving, option 2, manual, work
wget http://ftp.us.debian.org/debian/pool/main/libs/libseccomp/libseccomp2_2.5.2-2_armhf.deb  
sudo dpkg -i libseccomp2_2.5.2-2_armhf.deb

And it solve all !

**Reply by pkishino** [reply]
Yeah, raspberry had a number of problems with old binaries etc, good that you found the fix
